cypress/e2e/umd.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('web client recording spec', () => {
+	;['unpkg', 'jsdelivr'].map((source) => {
+		it('public graph requests are recorded', { baseUrl: null }, () => {
```

Comment:
Nit: could be nice to interpolate the `source` into this name incase one starts failing?

Reply:
good idea!